### PR TITLE
Only set the log level *if* bluetooth is available. 

### PR DESCRIPTION
**Pull request by efortuna**
Without this change, `setLogLevel` inside `FlutterBlue.instance` throws a PlatformException

### CODE REVIEW

**Comment by andretietz** [comment]
Please accept the CLA, so that someone can merge this. Or let us know if you don't want to, so I would create the PR.

**Comment by andretietz** [comment]
It doesn't seem to be in the master yet, so if there's a PR open for it already, please link it and ask @pauldemarco to close this PR.

**Comment by efortuna** [comment]
ah, hm. Okay. Let me get the CLA okay-ed by Google's lawyers since this change was done on company time. I'll get back with you shortly I hope!

**Comment by efortuna** [comment]
Hi @andretietz I got the approval from Google lawyers. Can you take another look?

**Comment by andretietz** [comment]
Looks good so far, I personally would remove the version change, since this is up to @pauldemarco to decide. If he merges something else before and increasing the version number, he will change this anyways.

File: pubspec.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: flutter_blue
 description:
   Flutter plugin for connecting and communicating with Bluetooth Low Energy devices,
   on Android and iOS
-version: 0.6.0+4
+version: 0.6.1
```
Comment (andretietz) [inline pubspec.yaml]:
```suggestion
version: 0.6.0+4
```

Comment (efortuna) [reply pubspec.yaml]: If I make this change do I remove the entry from the CHANGELOG.md too?

Comment (andretietz) [reply pubspec.yaml]: I would do so.

**Comment by efortuna** [comment]
made the requested changes

**Comment by andretietz** [comment]
@efortuna Did you check that it doesn't crash, if there's a log before setting the loglevel?

**Comment by pauldemarco** [comment]
Hi @efortuna @andretietz,

Sorry for the delay on this, I have some questions.

It was my intention that setting the log level should succeed whether or not Bluetooth is available. On future platforms, we may need to set verbosity during times when Bluetooth is unavailable.  

It looks like this will only error out if FlutterBlue is being used on an unsupported platform... something other than Android/iOS. Was this the case?

**Comment by efortuna** [comment]
It's been a long time since I encountered this issue so I could be wrong, but if memory serves, this happens if you run it on an Android or iOS simulator? IIRC you have to construct a FlutterBlue object to be able to check the "platform is supported" call (it's not a static method) so you *have* to construct an object to do that check. Hence this constructor dealing with the resulting crash of setting logging on an "unsupported platform" (aka a simulator).

**Comment by andretietz** [comment]
It's basically crashing when (for some reason) running on a device that doesn't support bluetooth. Usually this cannot happen, since the play store is not showing apps to devices without bluetooth if bluetooth is required, but in cases of manually installing an apk and the emulator, this just crashes.

**Comment by efortuna** [comment]
This happens every time you run on a simulator. Which is a very common occurrence if you're developing an app. And this _plugin_ shouldn't crash just because a user has run it on some "unsupported device" (perhaps they have bluetooth off?).

**Pull request merged**